### ISSUE TITLE
XLwings & Dash. Attempting to pre-load Excel before Dash callback to save load time.

### ISSUE THREAD

**Issue by karnrage** [xlwings/xlwings#2293]
Windows 10 Enterprise for local. Windows Server 2022 Standard for the server

3.10 Python, Microsoft 365 Apps for enterprise, 0.28.7 xlwings

I have a web interface project that is using a connection between XLwings and Dash. The Excel file is large and takes about 15 seconds to load, while the calc only takes 5 seconds. Since it is setup to run over the web, each client will have their own instance of Excel. XLwings has a way to handle this.
Currently the setup is that once the user enters their setup and clicks submit, only then does the connection start. 
If this was normal python, I could just start the connection earlier. However, with Dash, the only connection between the layout, and the function definition is the Dash callback.
The goal is to load the Excel file as soon as the page is loaded, while that is happening the user would enter their parameters, the time that takes to load would be saved and the result return time would be shortened by that time.
If I setup the connection as a variable or function, it doesn't look like you can pass these through the callback in anyway.
Project requirements are that the Excel and its VBA not be converted to Python/Dash.
Possible to launch one, or a few, Excels' on launch, and have XLwings grab the PID somehow? Would another Excel extension handle this?

```
Above this part is the layout setup for Dash
```
@callback(
     #many Output and Inputs. Removed most for clarity.
     Output(component_id='table', component_property='data'),
     Input(component_id='yearSlider', component_property='value'),
)
def finalFxn(
#many parameters. Removed most for clarity.
parameter1,
parameter2
):
   with xw.App(visible=True) as app:
        wb = app.books.open(r"C:\Users\mike\Downloads\modified3-31.xlsm")
#logic
# excel cells = parameters from web
#logic
# parameters to web = excel 
return results, graph, etc.
```

**Comment by fzumstein** [comment]
While your use case may sometimes be ok to use if there really is not other solution, it isn't really a what xlwings was made for. You should try OpenPyXL that can write the Excel file directly whithout being dependent on Excel being installed.

**Comment by karnrage** [comment]
Hello fzumstein,
With OpenPyXL, when you say you can write/read to the Excel file, that is only for the file itself?
I had the understanding that the file contains the data/VBA code, and Excel itself does the controlling of the calculations.

Are you saying the calculations can be run, and the inputs sent to the Excel file, and outputs returned from the file with OpenPyXL?
What I'm reading is that I can interact with the file directly, and OpenPyXL emulates Excel, and the program itself is not needed. Correct me if I'm wrong

**Comment by fzumstein** [comment]
Yes, your right, OpenPyXL is for reading and writing, it can't evaluate formulas AFAIK. If you need that, maybe xlwings is indeed better, but I don't have enough knowledge about Dash to be able to help.

**Comment by karnrage** [comment]
I was looking into xlWings pro, and it's limitations
-v0.28.0 says that it comes with an ultra fast file reader compared to pandas.read_excel()
-Says file reader only, no writing, but doesn't say about calculations/formulas. Can you confirm it doesn't do calcs?
-Also says cell values only, not cell formulas. What about VBA?
-Also, no table name support. So, if I have any calcs dependent on cell formulas and/or table name then it won't work, correct?

I can try the trial and see, this would be faster than trying to figure out the file.

**Comment by fzumstein** [comment]
Yes, this is a pure reader, no calcs, no formulas, no tables for now. All it has in common with the classic version of xlwings is the syntax.
